src/test_workflow/integ_test/service_opensearch.py

```diff
@@ -56,6 +58,9 @@ def start(self) -> None:
         self.process_handler.start(self.dist.start_cmd, self.install_dir)
         logging.info(f"Started OpenSearch with parent PID {self.process_handler.pid}")
 
+        ServiceOpenSearch.transport_stdout = self.process_handler.stdout_data
+        ServiceOpenSearch.transport_stderr = self.process_handler.stderr_data
```

Comment:
This sets a global value, should be on the instance, `self`.

Reply:
> Member

Makes sense, @dblock .. indeed, it will not be able to support multiple instances ... let me go back to work on the instence level solution. Thank you!!
As for the test case, I think #3034 may not be a bug but enhancement ( adding local-cluster process log when starting the cluster ) , let me also think how to improve the test cases for it ... thanks again!